BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -55,12 +63,21 @@ public struct ElementContent {
         }
     }
 
+    /// Using a tree of `LayoutResultNode` values, enumerates
+    /// all children to return their `ElementState`, `Element`,
+    /// and relevant `LayoutResultNode` values.
+    ///
+    /// ## Note
+    /// This method does _not_ invoke the `forEach`block for the initial reciever.
+    /// If you want to perform work on the reciever, do it before calling this method.
+    ///
+    /// Delegates to `storage` (`ContentStorage`).
     func forEachElement(
         in size: CGSize,
         with environment: Environment,
         children childNodes: [LayoutResultNode],
         state: ElementState,
-        forEach: (ElementState, Element, LayoutResultNode) -> Void
+        forEach: (ElementState, Element, LayoutResultNode, [LayoutResultNode]) -> Void
```

Comment:
This is a pretty unwieldy set of parameters; consider making this have one argument of a `struct`, so the parameters are named.

Reply:
Done